docs/internal_api/index.rst

```diff
@@ -155,12 +165,55 @@ Utils
 
    io.utils._parse_grid_type
 
-Geometry
---------
+Grid Parsing and Encoding
```

Comment:
We already have a Heading-1 "IO" right before this (at Line 151) and it had already started heading-2s "Ugrid" and "Utils". To keep docs in parallel with namespacing, it makes more sense to stick to those ones instead of creating a new title I think.